projects/ada4355_fmc/zed/README.md

```diff
@@ -1,8 +1,38 @@
-# ADA4355 Zed HDL Project
+<!-- no_build_example, no_dts, no_no_os -->
 
-## Building the project
+# ADA4355-FMC/ZED HDL Project
+
+- VADJ with which it was tested in hardware: 2.5V
+
+The parameters configurable through the `make` command, can be found below, as well as in the **system_project.tcl** file; it contains the default
+configuration.
 
 ```
-cd projects/ada4355/zed
+cd projects/ada4355_fmc/zed
 make
-```
\ No newline at end of file
+```
+
+This project supports two evaluation boards, which are differentiated by how the frame clock signals are distributed.
```

Comment:
Please write here the distinctive marks, so the users know which is an older board and which is a newer board.